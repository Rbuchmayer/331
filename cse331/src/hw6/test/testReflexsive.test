LoadGraph reflex reflexsiveGraph.tsv
ListChildren reflex A
ListChildren reflex B