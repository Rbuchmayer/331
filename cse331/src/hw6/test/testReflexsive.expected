loaded graph reflex
the children of A in reflex are: B(1) 
the children of B in reflex are: A(1) 
